### PR TITLE
Ready: Fix examples download cli cmd

### PR DESCRIPTION
**Pull request by lex-c**
Changed versioning scheme. The scm versioning in setup py was not working correctly. I substituted from param (as in datashader) to generate versions. Conda versions are fixed at 0.1.6 right now.

### CODE REVIEW

**Comment by lex-c** [comment]
After conda compatiblity and manifest changes, no longer downloads correctly with pip.

**Comment by lex-c** [comment]
@brendan This works now with conda and with pip, and it's by fixing the version. But, in order to make it a single-source version and to generate "dirty" local version names, the single source is the git refs tags, and I have not yet been able to verify that this will update correctly.

**Comment by lex-c** [comment]
@brendan this is ready to merge. I added a test script for the pip packaging and some little formatting lines in setup.py for the pip packaging. As long as a version with a new number is generated and that one is installed (and not the cached previous one) the examples download correctly.

**Comment by lex-c** [comment]
@brendancol This is ready to merge

**Comment by lex-c** [comment]
@brendancol This is ready to merge

File: setup.py [hunk 1]
```diff
@@ -42,15 +46,21 @@
...
 # metadata for setuptools
 
 setup_args = dict(
-    name='xarray-spatial',
-    use_scm_version={
-        'write_to': 'xrspatial/_version.py',
-        'write_to_template': '__version__ = "{version}"',
-        'tag_regex': r'^(?P<prefix>v)?(?P<version>[^\+]+)(?P<suffix>.*)?$',
-    },
+    name='xarray-spatial-chlochlo',
```
[the remark this answers is not in the document]
Comment (brendancol) [reply setup.py:66]: @lex-c hey looks like `chlochlo` need to be removed in the name argument

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -2,29 +2,29 @@
 
 package:
   name: xarray-spatial
-  version: 0.1.8
+  version: "0.1.8"
```
[the remark this answers is not in the document]
Comment (lex-c) [reply conda.recipe/meta.yaml:5]: Yes, I think it was doing that.

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lex-c) [reply setup.py:66]: I forgot to change that back; I'll fix that now.

**Review by brendancol — COMMENTED** [review]
@lex-c hey hey...looks like there is a conflict with meta.yaml.  I think it just requires to set it to `0.1.20` or accepting `theirs` in resolving the conflict
 ping me when the conflict is resolve and I'll merge

**Pull request merged**